### PR TITLE
Fix/dedicated user calendar

### PR DESCRIPTION
**Pull request by cbourgois**
### Description of the Change

ref: MANAGER-3843

1dead5946 — build(deps): update ovh-ui-angular
c538fcba4 — fix(dedicated): replace account calendar

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_eu#6242.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_ca#6242.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_us#6242.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_eu#6242.1 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_us#6242.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_ca#6242.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_us#6242.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_ca#6242.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

File: packages/manager/apps/dedicated/client/app/css/source.less [hunk 1]
```diff
@@ -1,14 +1,14 @@
 @oui-path: '../../../ovh-ui-kit';
 
+@import (less) "flatpickr/dist/flatpickr.min.css";
+@import (less) "flatpickr/dist/flatpickr.css";
```
Comment (marie-j) [inline packages/manager/apps/dedicated/client/app/css/source.less]: Why do you need to import both ?

File: packages/manager/apps/carrier-sip/package.json [hunk 2]
```diff
@@ -41,7 +41,7 @@
     "angular-ui-bootstrap": "~1.3.3",
     "angular-validation-match": "^1.9.0",
     "chart.js": "^2.8.0",
-    "flatpickr": "^4.6.3",
+    "flatpickr": "~4.5.2",
```
Comment (FredericEspiau) [inline packages/manager/apps/carrier-sip/package.json]: I think you meant `downgrade` instead of `upgrade` 😄

**Pull request merged**